packages/miniflare/src/workers/d1/dumpSql.ts

```diff
@@ -138,6 +147,15 @@ export function sqliteQuote(token: string) {
 		: token;
 }
 
+/**
+ * Escape an identifier for use in SQL statements.
+ * @param id - The identifier to escape.
+ * @returns
+ */
+function escapeId(id: string) {
```

Comment:
SQLite escapes double quotes in identifiers by doubling them using `sqlite3_mprintf("\"%w\"", azCol[i])`
https://github.com/sqlite/sqlite/blob/master/src/shell.c.in#L2914

The `%w` format doubles " as "":
https://github.com/sqlite/sqlite/blob/master/src/printf.c#L34